test/jdk/java/lang/reflect/OldenCompilingWithDefaults.java

```diff
@@ -25,8 +25,7 @@
  * @test
  * @bug 8009267
  * @summary Verify uses of isAnnotationPresent compile under older source versions
- * @compile -source 1.7 -target 1.7 OldenCompilingWithDefaults.java
- * @compile                         OldenCompilingWithDefaults.java
+ * @compile OldenCompilingWithDefaults.java
```

Comment:
I skimmed through the description of JDK-8009267 and I'm wondering if this test is of any value now.

Reply:
Certainly less useful than it once was!

I can remove it in the next push.